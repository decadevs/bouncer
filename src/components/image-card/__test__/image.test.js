import React from "react";
import { render } from "@testing-library/react";
import theme from "../../theme";
import { ThemeProvider } from "styled-components";
import ImageCard from "..";
import '@testing-library/jest-dom/extend-expect'

test("test to match snapshot", async () => {
    const { getByTestId } = render(
      <ThemeProvider theme={theme}>
        <ImageCard />
      </ThemeProvider>
    );
    const imageCard = getByTestId('image-card')
    expect(imageCard).toMatchSnapshot()
});
  