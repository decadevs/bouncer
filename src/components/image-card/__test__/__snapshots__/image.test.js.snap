// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test to match snapshot 1`] = `
<div
  class="d-flex justify-content-around"
  data-testid="image-card"
>
  <div
    class="sc-bdVaJa kvoOeF"
  >
    <img
      alt="different-views"
      src="https://res.cloudinary.com/decagonbouncer/image/upload/v1579183235/bouncer-frontend/side-view-beat-solo_nbuw8j.webp"
    />
  </div>
  <div
    class="sc-bdVaJa kvoOeF"
  >
    <img
      alt="different-views"
      src="https://res.cloudinary.com/decagonbouncer/image/upload/v1579183235/bouncer-frontend/beat-solo-partial-view_kkd1ga.webp"
    />
  </div>
  <div
    class="sc-bdVaJa kvoOeF"
  >
    <img
      alt="different-views"
      src="https://res.cloudinary.com/decagonbouncer/image/upload/v1579183235/bouncer-frontend/beat-solo-view_ayz1ye.webp"
    />
  </div>
  <div
    class="sc-bdVaJa kvoOeF"
  >
    <img
      alt="different-views"
      src="https://res.cloudinary.com/decagonbouncer/image/upload/v1579183119/bouncer-frontend/fold-beat-solo_b8hceq.webp"
    />
  </div>
</div>
`;
